ui/postcss2-index.js

```diff
@@ -0,0 +1,309 @@
+var __create = Object.create
+var __defProp = Object.defineProperty
+var __getProtoOf = Object.getPrototypeOf
+var __hasOwnProp = Object.prototype.hasOwnProperty
+var __getOwnPropNames = Object.getOwnPropertyNames
+var __getOwnPropDesc = Object.getOwnPropertyDescriptor
+var __markAsModule = (target) =>
+  __defProp(target, '__esModule', { value: true })
+var __export = (target, all) => {
+  for (var name in all)
+    __defProp(target, name, { get: all[name], enumerable: true })
+}
+var __exportStar = (target, module2, desc) => {
+  if (
+    (module2 && typeof module2 === 'object') ||
+    typeof module2 === 'function'
+  ) {
+    for (let key of __getOwnPropNames(module2))
+      if (!__hasOwnProp.call(target, key) && key !== 'default')
+        __defProp(target, key, {
+          get: () => module2[key],
+          enumerable:
+            !(desc = __getOwnPropDesc(module2, key)) || desc.enumerable,
+        })
+  }
+  return target
+}
+var __toModule = (module2) => {
+  return __exportStar(
+    __markAsModule(
+      __defProp(
+        module2 != null ? __create(__getProtoOf(module2)) : {},
+        'default',
+        module2 && module2.__esModule && 'default' in module2
+          ? { get: () => module2.default, enumerable: true }
+          : { value: module2, enumerable: true }
+      )
+    ),
+    module2
+  )
+}
+__markAsModule(exports)
+__export(exports, {
+  default: () => src_default,
+})
+var import_fs_extra = __toModule(require('fs-extra'))
+var import_util = __toModule(require('util'))
+var import_path = __toModule(require('path'))
+var import_tmp = __toModule(require('tmp'))
+var import_postcss2 = __toModule(require('postcss'))
+var import_postcss_modules = __toModule(require('postcss-modules'))
+var import_less = __toModule(require('less'))
+var import_stylus = __toModule(require('stylus'))
+var import_resolve_file = __toModule(require('resolve-file'))
+const postCSSPlugin = ({
+  plugins = [],
+  modules = true,
+  rootDir = process.cwd(),
+  sassOptions = {},
+  lessOptions = {},
+  stylusOptions = {},
+}) => ({
+  name: 'postcss2',
+  setup(build) {
+    let cache = new Map()
```

Comment:
Cache the css compilation to reduce the incremental build in the dev mode. Reduce the incremental build time from more than 4s to near 600ms.